qcelemental/models/common_models.py

```diff
@@ -44,10 +45,19 @@ def derivative_int(self):
 
 class ComputeError(ProtoModel):
     """The type of error message raised"""
-    error_type: str  # Error enumeration not yet strict
-    error_message: str
-    extras: Optional[Dict[str, Any]] = None
-
+    error_type: str = Schema(  # Error enumeration not yet strict
+        ...,
+        description="The type of error which was thrown. Restrict this field short classifiers e.g. 'input_error'"
```

Comment:
Could you please add the period to the end of this sentence? (Its something I missed in the example) Otherwise it looks good to me!